apps/api/src/app/events/events.controller.ts

```diff
@@ -86,6 +89,7 @@ export class EventsController {
 
   @ExternalApiAccessible()
   @UseGuards(UserAuthGuard)
+  @ApiBearerAuth()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** is this needed if we have the same annotation on the class level?